widgetssdk/src/main/java/com/glia/widgets/chat/ChatType.kt

```diff
@@ -3,8 +3,30 @@ package com.glia.widgets.chat
 import android.os.Parcelable
 import kotlinx.parcelize.Parcelize
 
+/**
+ * Used to identify which type of chat engagement to lunch.
+ *
+ * Example:
```

Comment:
For consistency with other files:
```suggestion
 * Code example:
```